samples/QR-app-installation/csharp/README.md

```diff
@@ -0,0 +1,124 @@
+﻿﻿﻿
+## Install app using barcode sample
+
+This sample demos app installation using QR code.
+
+The user can Generate a new QR code (contains app id information) and then use Install action to scan and install the app.
+
+`Currently, Microsoft Teams support for QR or barcode scanner capability is only available for mobile clients`
+
+![Card](QRAppInstallation/Images/CardWithButtons.png)
+
+![QR Code](QRAppInstallation/Images/QRCode.png)
```

Comment:
Please update the image with Polly app selected.

Reply:
done